packages/expo-splash-screen/ios/EXSplashScreen/EXSplashScreenViewNativeProvider.m

```diff
@@ -0,0 +1,30 @@
+// Copyright © 2018 650 Industries. All rights reserved.
+
+#import <EXSplashScreen/EXSplashScreenViewNativeProvider.h>
+#import <UMCore/UMLogManager.h>
+
+@implementation EXSplashScreenViewNativeProvider
+
+- (nonnull UIView *)createSplashScreenView:(EXSplashScreenImageResizeMode)resizeMode {
+  @try {
+    UIStoryboard *storyboard = [UIStoryboard storyboardWithName:@"SplashScreen" bundle:[NSBundle mainBundle]];
+    UIViewController *splashScreenViewController = [storyboard instantiateViewControllerWithIdentifier:@"SplashScreenViewController"];
+    UIView *splashScreenView = splashScreenViewController.view;
+    return splashScreenView;
+  } @catch (NSException *_) {
+    UMLogInfo(@"Expo SplashScreen.storyboard is missing.");
```

Comment:
If you think of the storyboard as _Expo_, I think this class should be named `EXSplashScreenViewExpoProvider` 🤔

Reply:
`EXSplashScreenViewExpoProvider` tells me nothing 🤔 My idea here is to have an `EXSplashScreenViewProvider` that bases on native `Splash Screen` that iOS presents upon application start.

Reply:
I see, if so, maybe we should remove the _Expo_ part from the log and describe that file some other way? (At least) for me it suggests that it is somehow strictly and magically governed by Expo and not by the user.